spectrum/gamestates/levelstate.lua

```diff
@@ -119,7 +82,19 @@ function LevelState:draw()
       secondary = {}
    end
 
-   self.display:drawPerspective(primary, secondary)
+   self.display:clear()
+   self:_draw(primary, secondary)
+   self.display:draw()
+end
+
+function LevelState:_draw(curActor, primary, secondary)
```

Comment:
Could we call this or the other method something else rather than going for underscores? Maybe move fetching the actor and building senses into their own functions?

Reply:
e.g.

```lua
function MyLevelState:draw()
   local primarySenses, secondarySenses = self:getSenses()

   self.display:clear()
   -- Do whatever
   self.display:draw()
end
```